### PR TITLE
Add -ListAvailable param to Get-PlasterTemplate

### PR DESCRIPTION
**Pull request by JustinGrote**
On the heels of #336, provide additional clarification and a parameter for "classic" operation due to slightly breaking change.

### CODE REVIEW

File: src/GetPlasterTemplate.ps1 [hunk 1]
```diff
@@ -36,10 +36,15 @@ function Get-PlasterTemplate {
         [Parameter(Position=0,
                    Mandatory=$true,
                    ParameterSetName="IncludedTemplates",
-                   HelpMessage="Initiates a search for Plaster templates inside of installed modules.")]
+                   HelpMessage="Initiates a search for Plaster templates inside of installed modules. By default, this retrieves from the latest version available for each module.")]
```
Comment (rkeithhill) [inline src/GetPlasterTemplate.ps1]: I'd prefer to keep this help message shorter - considering where it shows up.  But could you add this detail to the command's doc/md file in `docs/en-us/Get-PlasterTemplate.md`?  Same for the new parameter below.  Thanks!

File: src/GetPlasterTemplate.ps1 [hunk 2]
```diff
@@ -109,4 +120,4 @@ function Get-PlasterTemplate {
             }
         }
     }
-}
\ No newline at end of file
+}
```
Comment (rkeithhill) [inline src/GetPlasterTemplate.ps1]: 👍

File: src/GetPlasterTemplate.ps1 (hunk 1 above)
Comment (JustinGrote) [reply src/GetPlasterTemplate.ps1]: Sure I'll clean that up

File: docs/en-US/Get-PlasterTemplate.md [hunk 3]
```diff
@@ -19,15 +19,18 @@ Get-PlasterTemplate [[-Path] <String>] [[-Name] <String>] [[-Tag] <String>] [-Re
 
 ### IncludedTemplates
 ```
-Get-PlasterTemplate [-IncludeInstalledModules] [[-Name] <String>] [[-Tag] <String>] [<CommonParameters>]
+Get-PlasterTemplate [-IncludeInstalledModules] [-ListAvailable] [[-Name] <String>] [[-Tag] <String>] [<CommonParameters>]
```
Comment (rkeithhill) [inline docs/en-US/Get-PlasterTemplate.md]: Take a look at the Tag parameter section starting around line 169.  I think we need a section like that for this new parameter.  Or if you want, I can add that later.  Up to you.

Comment (rkeithhill) [reply docs/en-US/Get-PlasterTemplate.md]: To be honest, I need to refresh my memory on how this doc works with platyPS.  I think platyPS generates this once (up front) but maybe it updates it with parameter info on every build?

Comment (tylerl0706) [reply docs/en-US/Get-PlasterTemplate.md]: https://github.com/PowerShell/Plaster/blob/master/build.psake.ps1#L235-L317

It looks like all of the docs are generated at build time.

**Comment by rkeithhill** [comment]
@JustinGrote Thanks again!!

**Pull request merged**